.github/workflows/build-documentation.yml

```diff
@@ -30,6 +30,3 @@ jobs:
     - name: Install notebook requirements
       run: |
         sudo apt-get install pandoc
```

Comment:
If you set up read the docs correctly to build your docs then you shouldn't need this gh workflow anymore.

Reply:
Okay got it